composer.json

```diff
@@ -0,0 +1,34 @@
+{
+	"name": "hm-base",
+	"license": "GPL V3+",
+	"minimum-stability": "stable",
+	"config": {
+		"vendor-dir": "content/plugins-mu/vendor"
+	},
+	"repositories": [
+		{
+			"type": "composer",
+			"url": "https://wpackagist.org"
+		}
+	],
+	"require-dev": {
+		"phpunit/phpunit": "5.7",
```

Comment:
Do we need phpunit? If you're using Chassis Tester as an extension it should install this for you

Reply:
That's a fair point, I suppose we really don't need to include PHPUnit locally or on CI.

Reply:
It comes down to if a project wants to assume PHPUnit is present in all environments where it'll be needed -- or not.